src/helpers.js

```diff
@@ -154,9 +153,25 @@ export function normalizeSwagger(parsedSpec) {
               if (!operation[inhName]) {
                 operation[inhName] = inherits[inhName]
               }
+              else if (inhName === "parameters") {
+                var exists = false
+                for (const item in inherits[inhName]) {
+
+                  for (const param in operation[inhName]) {
+                    if (operation[inhName][param].name === inherits[inhName][item].name) {
+                      exists = true
+                    }
+                  }
+
+                  if (!exists) {
```

Comment:
Confirmed: above code logic is incorrect. This test will fail:

```
        const spec = {spec: {
          paths: {
            '/two': {
              parameters: [
                {name: 'a', in: 'path'},
                {name: 'b', in: 'path'}
              ],
              get: {
                parameters: [
                  {name: 'a', in: 'query'},
                  {name: 'c', in: 'query'}
                ]
              }
            }
          }
        }}

        const resultSpec = normalizeSwagger(spec)

        expect(resultSpec).toEqual({spec: {
          paths: {
            '/two': {
              parameters: [
                {name: 'a', in: 'path'},
                {name: 'b', in: 'path'}
              ],
              get: {
                parameters: [
                  {name: 'a', in: 'query'},
                  {name: 'b', in: 'path'}, // <---  'b' isn't pushed because `exists` isn't reset
                  {name: 'c', in: 'query'}
                ]
              }
            }
          }
        }})
```